Src/ILGPU.Tests/GridOperations.cs

```diff
@@ -12,7 +12,6 @@ public abstract class GridOperations : TestBase
         protected GridOperations(ITestOutputHelper output, TestContext testContext)
             : base(output, testContext)
         { }
-
```

Comment:
nit: accidentally removed a line?

Reply:
In the last PR, I pulled this file from the master branch but it still refuses to believe I didn't touch it...

Reply:
In order to fix this, what should I do? I can close the PR and push the branch again if necessary.